packages/api/prisma/seed-base.ts

```diff
@@ -0,0 +1,140 @@
+import {PrismaClient} from '@prisma/client'
```

Comment:
Documentation has to be updated now. This also makes a breaking change.

Reply:
That was something I wanted to discuss. Did that in the last moment as could not make seed run without example.